### DISCUSSION TITLE
Die Wlan-Rechte der anderen?

### DISCUSSION THREAD

**Discussion by fda77** [Freetz-NG/freetz-ng#917] [Allgemein]
Für AVM läuft es nicht überall so wie bei den "Labor" und den Quellcodepaketen die nicht herausgerückt werden.

Von Huawei gab es Klagen unter anderem gegen AVM
 - [Golem - Huawei verklagt AVM wegen WLAN-Patenten in Fritzbox](https://www.golem.de/news/netgear-amazon-huawei-verklagt-avm-wegen-wlan-patenten-in-fritzbox-2211-169621.html)
 - [Heise - Huawei v. AVM: Fritzbox-Vertriebsverbot angedroht, aber nicht rechtskräftig](https://www.heise.de/hintergrund/Fritzboxen-AVM-verliert-Patentklage-durch-Huawei-in-erster-Instanz-9575693.html)
 - [PCGH - Fritzbox: AVM droht Verkaufsverbot wegen Verstößen gegen WLAN-Patente von Huawei](https://www.pcgameshardware.de/Internet-Thema-34041/News/AVM-Fritzbox-Verbot-wegen-Verstoss-gegen-WLAN-Patente-1436400/)

**Comment by fda77** [comment]
Wie auf https://www.deskmodder.de/blog/2024/01/08/avm-hinweise-auf-patent-klage-verdichten-sich/ zu lesen gibt es mutmasslich einen Eintrag im Changelog von AVM dazu!
```
## WLAN:
- **Änderung** Anpassung von Signalisierungsfeldern bei Wi-Fi-6
```
Nachzulesen in Zeile 95(!!) von https://download.avm.de/fritzbox/fritzbox-6690-cable/deutschland/fritz.os/info_de.txt Falls diese Zeile wirklich neu ist

**Reply by Conan179** [reply]
https://www.heise.de/news/Fritzboxen-AVM-bringt-FritzOS-Updates-wegen-Huawei-Patentklage-9592057.html
"Reaktion auf verlorene Huawei-Klage
Auf Nachfrage bestätigte AVM, dass die Firmware-Updates [mit Huaweis Patentklage gegen die Wi-Fi-6-Fritzboxen zusammenhängen](https://www.heise.de/hintergrund/Fritzboxen-AVM-verliert-Patentklage-durch-Huawei-in-erster-Instanz-9575693.html)."

**Reply by fda77** [reply]
Schön dass AVM endlich gesagt hat wozu die Updates gut sind

**Comment by fda77** [comment]
[Golem: Huawei will 50-Cent-Lizenzgebühr pro AVM-Fritzbox](https://www.golem.de/news/verkaufsverbot-hauwei-will-50-cent-lizenzgebuehr-pro-avm-fritzbox-2401-181083.html)
